### PR TITLE
Porfolio

### PR DESCRIPTION
**Pull request by valt19**
# portfolio 
In this project:
add linters
add html
styled the cards
updated the README file

### CODE REVIEW

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @valt19, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
**Congratulations 🎉 on the impressive work you've done so far. I've made some suggestions below to help you improve your project so kindly take a look at it and if you don't understand anything please don't hesitate to ask. Best of luck! 🤞🏻. Using `inline` comments would have been the best but since I don't have access to your `index.html` in this pull request. I will do my best to explain the changes here. Let's get started.**

- [ ] Consider removing the `welcome` text you have on the `nav` as it's not on the `Figma` template design. This will make your design look accurate and look exactly like that of the `Figma` design template. See below for more details:
<table>
<tr>
<th align="middle">Figma design</th>
<th align="middle">Your design</th>
</tr>
<tr>
<td>
Screenshot 2023-04-28 at 04 44 59
</td>
<td>
Screenshot 2023-04-28 at 11 30 11
</td>
</tr>
</table>

- [ ] Consider moving the Iine break `br` in your `h1` between **Hey there** and  **I’m Mariane**. This way the text  **I’m Mariane** will be wrapped onto the next line and that will make your design look exactly like that of the `Figma` design.
https://github.com/valt19/porfolio-/blob/f31491cd0801972b57359ad3c1b5bac7db566183/index.html#L21

- [ ] Kindly consider the use of heading (`h2`, `h3`, `h4`) tags instead of the `span` tag to display the sub-heading of the main heading. Using heading tags conveys the semantic meaning of a subheading and helps improve the accessibility and structure of the document. Therefore, using heading tags instead of the `span` tag would be more semantically correct in this context.
https://github.com/valt19/porfolio-/blob/f31491cd0801972b57359ad3c1b5bac7db566183/index.html#L22

- [ ] Consider including `alt` text for your images. Including `alt` text in their images is not only good for accessibility, but it's also an essential part of good web development practice. The `alt` attribute is used to describe the contents of an image to users who may not be able to see the image. This includes users with visual impairments, users who use screen readers, and users who may have slow internet connections or have turned off image loading to save bandwidth.

https://github.com/valt19/porfolio-/blob/f31491cd0801972b57359ad3c1b5bac7db566183/index.html#L27-L46

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -4,276 +4,209 @@
...
-        <nav class="menu-bar-desktop">
-          <ul class="nav-menu-desktop">
-            <!-- <a href="" onclick="" class="close-button">&times;</a> -->
-            <li ><a href="#portfolio">Portfolio</a></li>
-            <li><a href="#about">About</a></li>
-            <li><a href="#contact">Contact</a></li>
-          </ul>
-        </nav>
-        
+      <!-- <a href="welcome" class="head">Welcome</a> -->
```
Comment (jerryowusu) [inline index.html]: Nice work so far. However, as part of HTML/CSS best practices, do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.
Kindly consider removing the commented line of code from your codebase. :pray:

File: index.html [hunk 2]
```diff
@@ -4,276 +4,209 @@
...
+        <img src="images/Normal Button/Enabled.png" alt=" menu-bar">
+      </nav>
     </header>
-    
     <section class="head-section">
     <div class="hey-there">
-<h1 class="hey-text">Hey there. I’m Mariane <br>
-  <span class="hey-text2"> I’m a software developer</span></h1>
+      <h2 class="hey-text">Hey there.<br> I’m Trevor.<br>
+      <span class="hey-text2"> I’m a software developer</span></h2>
```
Comment (jerryowusu) [inline index.html]: [OPTIONAL] Kindly make sure that this part looks exactly like the design in all screen sizes of the mobile version. There should be a line break between the `I'm a software` and `developer`, however, at screenwidths greater than 600px, they stay on the same line. Kindly update your app accordingly. :pray: 

### Your app below 600px 

Screenshot from 2023-04-29 08-28-48


### Your app above 600px

Screenshot from 2023-04-29 08-27-20



### Actual design 


Screenshot from 2023-04-29 08-25-55

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @valt19  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

soclose2


 ## Highlights
Nice design so far :+1: 
Great effort in making the requested changes :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @valt19  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**